src/utils/find.ts

```diff
@@ -123,11 +124,13 @@ function findAllVNodes(
         aggregateChildren(nodes, fallbackTree.children)
       }
     }
-    if (matches(node, selector)) {
+    if (matches(node, selector) && matchingNodes.indexOf(node) < 0) {
       matchingNodes.push(node)
     }
   }
 
+  // console.log(matchingNodes)
```

Comment:
Should we maybe get rid of this before merging?

Reply:
Yep definitely, good to keep things clean. I think he will clean the code a little before we merge it up.

I will do a release once this is merged!